docs/middleware/ics29-fee/end-users.md

```diff
@@ -0,0 +1,31 @@
+<!--
+order: 6
+-->
+
+# For End Users
+
+Learn how to incentivize IBC packets using the ICS29-Fee module. {synopsis}
+
+## Pre-requisite Readings
+
+* [Fee Middleware](overview.md) {prereq}
+
+## Summary
+
+Different types of end users:
+
+- CLI Users who want to manually incentivize IBC packets
+- Client Developers
+
+
+The Fee Middleware module allows end users to add a 'tip' to each IBC packet which will incentivize relayer operators to relay packets between chains. GRPC endpoints are exposed for client developers as well as a simple CLI for manually incentivizing IBC packets.
+
+## CLI Users
+
+For an in depth guide on how to use the ICS29 Fee module using the CLI please take a look at the [wiki](https://github.com/cosmos/ibc-go/wiki/Fee-enabled-fungible-token-transfers#asynchronous-incentivization-of-a-fungible-token-transfer) on the `ibc-go` repo.
```

Comment:
I think we can keep this section simple as there's no point in re-creating the wheel. Most people won't use the CLI anyway.